### PR TITLE
Added episode data from tmdb

### PR DESCRIPTION
**Pull request by PabloGrelaM**
## Summary by CodeRabbit

- **New Features**
  - Added functionality to display detailed episode ratings and images in the Character Details section.
  - Introduced new UI elements in the Character Details screen for displaying episode details.
  
- **Enhancements**
  - New methods and data handling for fetching and displaying episode details from TMDB API.

### CODE REVIEW

**Comment by PabloGrelaM** [comment]
This PR completes #24

File: app/src/main/java/com/mstudio/superheroarch/CharacterDetailsActivity.kt [hunk 1]
```diff
@@ -42,6 +42,13 @@ class CharacterDetailsActivity : AppCompatActivity(), CharacterDetailsViewTransl
         findViewById<TextView>(R.id.detailsAirDateTextView).text = episode.releaseDate
     }
 
+    override fun showEpisodeDetails(episode: TMDBEpisodeData) {
+        findViewById<TextView>(R.id.detailsRatingTextView).text = episode.rating.toString()
+        Glide.with(this)
+            .load(episode.imagePath)
+            .into(findViewById(R.id.detailsHeaderImage))
+    }
```
[the remark this answers is not in the document]
Comment (PabloGrelaM) [reply app/src/main/java/com/mstudio/superheroarch/CharacterDetailsActivity.kt:50]: What do you suggest?

File: app/src/main/res/layout/activity_character_details.xml [hunk 2]
```diff
@@ -12,15 +12,27 @@
...
 
+    <ImageView
+        android:id="@+id/detailsHeaderImage"
+        android:contentDescription="@string/details_header_image"
+        app:layout_constraintDimensionRatio="16:9"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        android:layout_width="0dp"
+        android:layout_height="0dp"/>
```
[the remark this answers is not in the document]
Comment (PabloGrelaM) [reply app/src/main/res/layout/activity_character_details.xml:23]: What do you suggest?

[the remark this answers is not in the document]
Comment (PabloGrelaM) [reply app/src/main/res/layout/activity_character_details.xml:23]: Makes no sense, I'm keeping a full width with 16:9 aspect ratio. There are no margins

File: app/src/main/java/com/mstudio/superheroarch/CharacterDetailsActivity.kt (hunk 1 above)
[the remark this answers is not in the document]
Comment (PabloGrelaM) [reply app/src/main/java/com/mstudio/superheroarch/CharacterDetailsActivity.kt:50]: I think that's overkill as far as the showEpisodeDetails will be called always after the started state.

File: app/src/main/res/layout/activity_character_details.xml [hunk 3]
```diff
@@ -159,6 +182,28 @@
...
+
+    <TextView
+        android:id="@+id/detailsRatingTextView"
+        android:layout_width="wrap_content"
+        app:layout_constraintTop_toTopOf="@id/detailsRatingLabel"
+        app:layout_constraintBottom_toBottomOf="@id/detailsRatingLabel"
+        app:layout_constraintStart_toEndOf="@id/detailsRatingLabel"
+        android:layout_marginStart="16dp"
+        tools:text="8.3"
+        android:layout_height="wrap_content"/>
```
[the remark this answers is not in the document]
Comment (PabloGrelaM) [reply app/src/main/res/layout/activity_character_details.xml:205]: What a useless comment.

**Review by andreslopezsarmiento — DISMISSED** [review]

**Pull request merged**